src/main/java/org/gridsuite/study/server/networkmodificationtree/entities/RootNodeInfoEntity.java

```diff
@@ -56,4 +60,9 @@
     @Column(name = "buildStatus", nullable = false)
     @Enumerated(EnumType.STRING)
     private BuildStatus buildStatus;
+
+    @Column(name = "modificationsToExclude")
```

Comment:
Since no modifications are associated to Root node, is it necessary to add this column ?

Reply:
Same